R/layers2traces.R

```diff
@@ -844,6 +955,7 @@ geom2trace.GeomText <- function(data, params, p) {
     customdata = data[["customdata"]],
     frame = data[["frame"]],
     ids = data[["ids"]],
+    textposition = if("textposition" %in% names(data)) data[[1, "textposition"]] else NULL,
```

Comment:
```suggestion
    textposition = data$textposition[[1]],
```

Reply:
is `textposition` always provided in the `data` argument?

Reply:
I'm not assuming that, but it doesn't matter since `data$textposition[[1]]` in that case is still `NULL`